server/index.ts

```diff
@@ -11,11 +12,7 @@ app.use(express.json());
 app.use(cors());
 app.use(router);
 
-mongoose
```

Comment:
I have moved al db connection logic to its own file on model/db and added dotenv